chromium_src/chrome/browser/resources/tab_search/tab_search_api_proxy.ts

```diff
@@ -12,6 +12,7 @@ export interface BraveTabSearchApiProxy extends TabSearchApiProxy {
   getFocusTabs: (topic: string) => Promise<{ windowCreated: boolean, error: Error | null }>
   undoFocusTabs: () => Promise<void>
   openLeoGoPremiumPage: () => void
+  enableTabFocus: (enable: boolean) => void
```

Comment:
nit: maybe this should be called `SetTabFocusEnabled`, as `EnableTabFocus` sounds like its just going to enable it.

Reply:
Update the front-end variable name to `setTabFocusEnabled`

Reply:
Backend name is updated as well.